docs/en/sql-reference/functions/math-functions.md

```diff
@@ -711,3 +711,41 @@ Result:
 │                               11 │
 └──────────────────────────────────┘
 ```
+
+## byteSwap
+
+Accepts an unsigned integer `operand` and returns the integer which is obtained by swapping the **endianness** of `operand` i.e. reversing the bytes of the `operand`.
+
+Currently, this is supported for up to 64-bit (signed and unsigned) integers.
+
+**Example**
+
+```sql
+byteSwap(3351772109)
+```
+
+Result:
+
+```result
+┌─byteSwap(3351772109)─┐
+│           3455829959 │
+└──────────────────────┘
+```
+
+The above example can be worked out in the following manner:
+1. First, convert the integer operand (base 10) to its equivalent hexadecimal interpretation (base 16) in big-endian format i.e. 3351772109 -> C7 C7 FB CD (4 bytes)
+2. Then, reverse the bytes i.e. C7 C7 FB CD -> CD FB C7 C7
+3. Finally, the convert the hexadecimal number back to an integer assuming big-endian i.e. CD FB C7 C7  -> 3455829959
+
+Note that, in step#1, one can also choose to convert the operand to bytes in little-endian as long as one also assumes little-endian when converting back to integer in step#3.
+
+One use-case of this function is reversing IPv4s:
+```result
+┌─toIPv4(3351772109)─┐
```

Comment:
You probably want to include an example where an IPv4 is casted to an integer, which is then `byteSwap`ed, and casted to an IPv4 again.

Reply:
> Do you mean that instead of the two statements, we can just demonstrate reversing IPv4s with a single statement like the following

Done